susemanager-utils/susemanager-sls/src/_states/uyuni_users.py

```diff
@@ -0,0 +1,459 @@
+# coding: utf-8
+"""
+Uyuni users state module
+"""
+from typing import Any, Dict, List, Optional, Union, Tuple
+import logging
+from salt.modules.uyuni import UyuniUsersException, UyuniUser, UyuniOrg, RPCClient, UyuniTrust
+
+
+log = logging.getLogger(__name__)
+__pillar__: Dict[str, Any] = {}
+__salt__: Dict[str, Any] = {}
+__context__: Dict[str, Any] = {}
+__virtualname__: str = "uyuni"
+
+
+class UyuniFunctions:
+    """
+    RPC client
+    """
+    def __init__(self):
+        self.client = RPCClient.init(pillar=__pillar__)
+        self.users = UyuniUser(pillar=__pillar__)
+        self.orgs = UyuniOrg(pillar=__pillar__)
+
+    @staticmethod
+    def _get_proto_ret(name: str, result: Optional[bool] = None) -> Dict[str, Any]:
+        """
+        State proto return container.
+
+        :param name:
+        :return:
+        """
+        changes: Dict[str, str] = {}
+        return {
+            'name': name,
+            'changes': changes,
+            'result': result,
+            'comment': "",
+        }
+
+
+class UyuniOrgs(UyuniFunctions):
+    """
+    Uyuni operations over orgs and trusts.
+    """
+    ADMIN_PREFIXES = ["Dr.", "Hr.", "Miss", "Mr.", "Mrs.", "Ms.", "Sr."]
```

Comment:
Shouldn't we just get rid of those prefixes on the Uyuni side? No-on else is doing them!